nan.h

```diff
@@ -598,25 +598,33 @@ class TryCatch {
 # define NAN_GC_CALLBACK(name)                                                 \
     void name(v8::Isolate *isolate, v8::GCType type, v8::GCCallbackFlags flags)
 
+#if NODE_MODULE_VERSION <= NODE_4_0_MODULE_VERSION
+  typedef v8::Isolate::GCEpilogueCallback GC_EPILOGUE_CALLBACK_TYPE;
+  typedef v8::Isolate::GCPrologueCallback GC_PROLOGUE_CALLBACK_TYPE;
```

Comment:
Please use `CamelCase` for (new) typedefs. (`UPPER_CASE` types had an earlier life as macros and have not been renamed... yet). Oh, and maybe drop the `_TYPE` suffix. Just use the original names.